src/main/scala/org/deepdive/ddlog/DeepDiveLogCompiler.scala

```diff
@@ -346,7 +346,7 @@ class QueryCompiler(cq : ConjunctiveQuery, ss: CompilationState) {
       case _ => s" AS column_${cq.headTerms indexOf e}"
     }
     case ViewAlias => s" AS column_${cq.headTerms indexOf e}"
-    case UseVariableAsAlias => s""" AS "${DeepDiveLogPrettyPrinter.print(e)}""""
+    case UseVariableAsAlias => s""" AS "${DeepDiveLogPrettyPrinter.print(e)}"""" //"
```

Comment:
I tried both, and found with '"' the code a bit messy, and the current one looks cleaner imo...